app/javascript/src/hyacinth_v1/components/controlled_vocabularies/ControlledVocabularyShow.js

```diff
@@ -108,12 +107,18 @@ function ControlledVocabularyShow() {
       <div className="m-2">
         <Form className="term-search-form" onSubmit={onSearchHandler}>
           <InputGroup>
-            <TextInput sm={8} value={searchTerms} onChange={v => setSearchTerms(v)} placeholder="Search for terms..." />
-            <SearchButton onClick={onSearchHandler} />
-            { searchTerms.length > 0 && searchTerms.length < 3 && <Form.Text className="text-muted pl-2">Query must be at least three characters.</Form.Text> }
+            <Form.Control
+              type="text"
+              value={searchTerms}
+              onChange={(e) => setSearchTerms(e.target.value)}
+              placeholder="Search for terms..."
+            />
+            <SearchButton sm="auto" onClick={onSearchHandler} />
           </InputGroup>
         </Form>
 
+        { query.length > 0 && query.length < 3 && <Form.Text className="text-muted pl-2">Query must be at least three characters.</Form.Text> }
```

Comment:
```suggestion
        { searchTerms.length > 0 && searchTerms.length < 3 && <Form.Text className="text-muted pl-2">Query must be at least three characters.</Form.Text> }
```

Reply:
Good catch, thank you!  Missed that.  That was one of five files that had merge conflicts before I did the rebase.